template.tpl

```diff
@@ -763,7 +1068,7 @@ ___TEMPLATE_PARAMETERS___
         "defaultValue": false,
         "simpleValueType": true,
         "name": "userAgentEnrichment",
-        "checkboxText": "Enable client side user agent enrichment",
+        "checkboxText": "(Legacy) Enable client side user agent enrichment",
```

Comment:
i think the nested config here to pick which user properties to enrich are not useful. i assume most won't care if it's using client-side or server-side and end up using whatever the default is. for the very few that would care, I'm certain they'll select all. that said i think we can remove the nested config because it's too much distraction. we can bring it back if we get enough demand.

Reply:
So just remove this option and all the related group?

Reply:
that's what i think. although i acknowledge that my opinion here is a bit extreme, so totally understandable if folks do not agree

Reply:
oops to clarify y suggsetion, keep the top level config, but only remove the nested ones

Reply:
@yuhao900914 You just added these right? Is there specific customer demand for granular clients-side UA enrichment?

Reply:
No. But there already has some complaints related to some event / property changes while we upgrading the internal SDKs. 
There is no way to easily implemented the plugin in GTM, and it's a breaking change for the customers without data/ add on cannot update the name of event/event properties.

Reply:
So our team determined to make the template backward compatible and disable this by default and also callout in the template doc.

Reply:
Given this is marked as legacy, I don't think it impacts growth team's efforts since new customers are likely to skim over it.